aws/resource_aws_route53_record.go

```diff
@@ -101,6 +101,12 @@ func resourceAwsRoute53Record() *schema.Resource {
 							Type:      schema.TypeString,
 							Required:  true,
 							StateFunc: normalizeAwsAliasName,
+							DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+								if strings.ToLower(old) == strings.ToLower(new) {
+									return true
+								}
+								return false
```

Comment:
The whole function can be reduced to a single line:

```go
return strings.ToLower(old) == strings.ToLower(new)
```

😉

Reply:
💯  good catch